### ISSUE TITLE
Multicast Support

### ISSUE THREAD

**Issue by ChrisEineke** [tonyg/udp-exchange#4]
Hello Tony,
Would you consider to add multicast support?

**Comment by tonyg** [comment]
Yes, but I don't think it's trivial. It would need a carefully thought-through design. Thoughts and suggestions welcome.
